opba-api-security/src/main/java/de/adorsys/opba/api/security/service/impl/RsaJwtsVerifyingServiceImpl.java

```diff
@@ -0,0 +1,54 @@
+package de.adorsys.opba.api.security.service.impl;
+
+import de.adorsys.opba.api.security.domain.SignData;
+import de.adorsys.opba.api.security.service.RequestVerifyingService;
+import de.adorsys.opba.api.security.service.SignatureParams;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.JwtParser;
+import io.jsonwebtoken.Jwts;
+import lombok.extern.slf4j.Slf4j;
+
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.PublicKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.X509EncodedKeySpec;
+import java.util.Base64;
+
+@Slf4j
+public class RsaJwtsVerifyingServiceImpl implements RequestVerifyingService {
+    private static final String ALGORITHM = SignatureParams.ALGORITHM_RSA.getValue();
```

Comment:
ALGORITHM_RSA should be autowired from the configuration properties

Reply:
done